public/test/jest-setup.ts

```diff
@@ -23,6 +23,15 @@ angular.module('grafana.routes', ['ngRoute']);
 jest.mock('app/core/core', () => ({}));
 jest.mock('app/features/plugins/plugin_loader', () => ({}));
 
+/* Dynamic import in <Icon /> throws SyntaxError: Cannot use import statement outside a module in test environment.
```

Comment:
Mocking Icon component globally.

Reply:
This is temporary solution. It will be either removed if we decide to only import to Grafana icons that we use, or we update jest config so it won't throw errors for .js files.